presto-main/src/main/java/com/facebook/presto/sql/planner/RowExpressionVariableInliner.java

```diff
@@ -26,27 +26,27 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkState;
 
-public final class RowExpressionSymbolInliner
+public final class RowExpressionVariableInliner
         extends RowExpressionRewriter<Void>
 {
     private final Set<String> excludedNames = new HashSet<>();
-    private final Map<Symbol, Symbol> mapping;
+    private final Map<VariableReferenceExpression, RowExpression> mapping;
 
-    private RowExpressionSymbolInliner(Map<Symbol, Symbol> mapping)
+    private RowExpressionVariableInliner(Map<VariableReferenceExpression, RowExpression> mapping)
```

Comment:
nit: Why the mapping is to `RowExpression` (instead of `VariableReferenceExpression` ? :) )

Reply:
What inliner does is to replace variable reference with the value it assigned to (thus not limiting to VariableReference).

Reply:
@hellium01 : I see. So it can also be an `ConstantExpression`? Basically it has more power than the original `RowExpressionSymbolInliner` now ?

Reply:
`ExpressionSymbolInliner` today also inlines Expression other than SymbolReference :)